owncloud/ios-app
# [fix/passcode-settings-ui-update] Passcode Settings UI Update

## Description
Setting the completion handler was missing, which causes that the UI was not updated after enabling/disabling passcode lock

## Related Issue
#923 

## Motivation and Context
UI feedback to the user, after setting changed

## How Has This Been Tested?
- enable passcode
- disable passcode

As a result the settings UI should be updated immediately

## Screenshots (if appropriate):

## Types of changes


## Checklist:



## QA

- [X] (1) Biometrical disabled after entering passcode https://github.com/owncloud/ios-app/pull/926#issuecomment-797492878 [FIXED]
- [X] (2) Face ID is requested after disabling passcode https://github.com/owncloud/ios-app/pull/926#issuecomment-800028356 [FIXED]
